libs/api-hjarntorget/lib/apiHjarntorget.ts

```diff
@@ -403,20 +403,26 @@ export class ApiHjarntorget extends EventEmitter implements Api {
     const mvghostRequestBody = extractMvghostRequestBody(initBankIdResponseText)
 
     console.log("picking auth server???")
-    const mvghostResponse = await this.fetch('pick-mvghost', mvghostUrl, {
+
+    let mvghostResponse = await this.fetch('pick-mvghost', mvghostUrl, {
       redirect: 'follow',
       method: 'POST',
       body: mvghostRequestBody,
       headers: {
-        'Content-Type': 'application/x-www-form-urlencoded'
+        'Content-Type': 'application/x-www-form-urlencoded',    
       }
     })
 
     console.log("start bankid sign in")
     // We may get redirected to some other subdomain i.e. not 'm00-mg-local':
     // https://mNN-mg-local.idp.funktionstjanster.se/mg-local/auth/ccp11/grp/other
-
-
+    console.log("waiting abit so next call doesn't redirect back to 'other' instead of 'verify'")
+    // Need a short wait otherwise we end up in the wrong place!
+    const waitabit = new Promise((resolve) => {
+      setTimeout(() => resolve('foo'), 500)
+    });
+    await waitabit
+    console.log("done waiting....")
```

Comment:
This was a bit strange, but without a slight delay before the request to `https://mNN-mg-local.idp.funktionstjanster.se/mg-local/auth/ccp11/grp/other/snn` I got a redirect back to `https://mNN-mg-local.idp.funktionstjanster.se/mg-local/auth/ccp11/grp/other` instead of  `https://m05-mg-local.idp.funktionstjanster.se/mg-local/auth/ccp11/grp/verify`.